### PR TITLE
NxColorPicker validation - RSC-1091

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-1091

### CODE REVIEW

File: lib/src/components/NxColorPicker/__tests__/NxColorPicker.test.tsx [hunk 1]
```diff
@@ -4,154 +4,130 @@
...
+  it('does nothing when an input is clicked with no onChange prop', async function() {
+    const user = userEvent.setup(),
+        view = quickRender(),
+        radio = view.getByRole('radio', { name: /turquoise/i });
 
-    component.find('input').filterWhere(input => input.prop('value') === 'turquoise').simulate('change');
-
-    expect(onChange).toHaveBeenCalledWith('turquoise');
+    await user.click(radio);
   });
```
Comment (cindyko226) [inline lib/src/components/NxColorPicker/__tests__/NxColorPicker.test.tsx:85]: Is this intended to have no expect?

Comment (rpokorny) [reply lib/src/components/NxColorPicker/__tests__/NxColorPicker.test.tsx:85]: > Is this intended to have no expect?

Yes, it simply tests that no exception is thrown

Comment (cindyko226) [reply lib/src/components/NxColorPicker/__tests__/NxColorPicker.test.tsx:85]: Out of curiosity, what might be the exception if there is no expectation?

Comment (rpokorny) [reply lib/src/components/NxColorPicker/__tests__/NxColorPicker.test.tsx:85]: > Out of curiosity, what might be the exception if there is no expectation?

The purpose of this test is just to verify that `onChange` is in fact optional and that the component doesn't blow up if you don't specify it.

Comment (cindyko226) [reply lib/src/components/NxColorPicker/__tests__/NxColorPicker.test.tsx:85]: I see, thanks for the explanation.

**Review by cindyko226 — APPROVED** [review]

**Pull request merged**